<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSymbolPad</class>
 <widget class="QWidget" name="FormSymbolPad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(150, 150, 150);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="1" column="0">
    <widget class="QToolButton" name="toolButton_00">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>,</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="toolButton_01">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="toolButton_02">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QToolButton" name="toolButton_03">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>!</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QToolButton" name="toolButton_04">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="toolButton_05">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QToolButton" name="toolButton_PageUp">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="keypad.qrc">
       <normaloff>:/new/keypad/picture/keypad/triangle-up.png</normaloff>:/new/keypad/picture/keypad/triangle-up.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="toolButton_06">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>@</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="toolButton_07">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="toolButton_08">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QToolButton" name="toolButton_09">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>&amp;&amp;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QToolButton" name="toolButton_10">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>^</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QToolButton" name="toolButton_11">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>~</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QLabel" name="label_CurPage">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string>1/3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="toolButton_12">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>'</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QToolButton" name="toolButton_13">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>&quot;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QToolButton" name="toolButton_14">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="toolButton_15">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QToolButton" name="toolButton_16">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <widget class="QToolButton" name="toolButton_17">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>#</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QToolButton" name="toolButton_PageDown">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="keypad.qrc">
       <normaloff>:/new/keypad/picture/keypad/triangle-down.png</normaloff>:/new/keypad/picture/keypad/triangle-down.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QToolButton" name="toolButton_Return">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="keypad.qrc">
       <normaloff>:/new/keypad/picture/keypad/f0e2.png</normaloff>:/new/keypad/picture/keypad/f0e2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="QToolButton" name="toolButton_Space">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>空格</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QToolButton" name="toolButton_BackSpace">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="keypad.qrc">
       <normaloff>:/new/keypad/picture/keypad/backspace.png</normaloff>:/new/keypad/picture/keypad/backspace.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QToolButton" name="toolButton_Enter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="text">
      <string>回车</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="7">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">border-width:0px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="toolButton_CHS">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
        </property>
        <property name="text">
         <string>中文</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_ENG">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
        </property>
        <property name="text">
         <string>英文</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="autoRaise">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton_Math">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">border-width:1px;  
border-style:solid;
border-color: rgb(150, 150, 150);  </string>
        </property>
        <property name="text">
         <string>数学</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="keypad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>toolButton_CHS</sender>
   <signal>clicked()</signal>
   <receiver>FormSymbolPad</receiver>
   <slot>onCHS()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_ENG</sender>
   <signal>clicked()</signal>
   <receiver>FormSymbolPad</receiver>
   <slot>onENG()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>493</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>496</x>
     <y>76</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_Math</sender>
   <signal>clicked()</signal>
   <receiver>FormSymbolPad</receiver>
   <slot>onMath()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_PageUp</sender>
   <signal>clicked()</signal>
   <receiver>FormSymbolPad</receiver>
   <slot>onPrevPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>844</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>744</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton_PageDown</sender>
   <signal>clicked()</signal>
   <receiver>FormSymbolPad</receiver>
   <slot>onNextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>767</x>
     <y>272</y>
    </hint>
    <hint type="destinationlabel">
     <x>746</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPrevPage()</slot>
  <slot>onNextPage()</slot>
  <slot>onCHS()</slot>
  <slot>onENG()</slot>
  <slot>onMath()</slot>
 </slots>
</ui>
